openapi.yml

```diff
@@ -3512,6 +3512,11 @@ paths:
           in: query
           name: created_after
           description: Restricts results to only those that were created after the provided date
+        - schema:
+            type: boolean
+          in: query
+          name: needs_confirmation
+          description: "A confirmed `Registration` is one with an associated `Invite`. This filter returns those without an `Invite` when true, and those with an `Invite` when false."
```

Comment:
I think it reads better when you start with the condition and then the result.

Consider this approach:
"When true, the filter returns invites that have an associated registration. When false the filter returns invites without an associated registration."